root/etc/s6-overlay/s6-rc.d/init-certbot-config/run

```diff
@@ -23,6 +23,13 @@ for i in "${SANED_VARS[@]}"; do
     export echo "${i}"="$(echo "${!i}" | tr '[:upper:]' '[:lower:]')"
 done
 
+# Check for and install requested DNS plugins
+if grep -q "universal-package-install" <<< "${DOCKER_MODS}" && grep -q "certbot-dns" <<< "${INSTALL_PIP_PACKAGES}"; then
+    echo "**** Installing requested dns plugins ****"
+    /etc/s6-overlay/s6-rc.d/init-mod-universal-package-install-add-package/run
+    /etc/s6-overlay/s6-rc.d/init-mods-package-install/run
```

Comment:
```suggestion
    /etc/s6-overlay/s6-rc.d/init-mod-universal-package-install-add-package/run
    /etc/s6-overlay/s6-rc.d/init-mods-package-install/run
    echo "#!/usr/bin/with-contenv bash" >/etc/s6-overlay/s6-rc.d/init-mod-universal-package-install-add-package/run
    echo "#!/usr/bin/with-contenv bash" >/etc/s6-overlay/s6-rc.d/init-mods-package-install/run
```
Replacing the `run` scripts with empty (shebang only) files after this point would prevent the double run.

Reply:
Just tested this change and it breaks when other mods (that run during the normal mod run later in the init) try to add packages to be installed (ex: swag auto reload tries to install inotify) and causes a loop in the init. Going to have to revert https://github.com/linuxserver/docker-swag/pull/460/commits/bf21a51e1079d762b6817e855463c74c1a843aaf and just let it run twice. At least it's tested, sorry for the extra git work.